fault.h

```diff
@@ -27,16 +27,13 @@
 /**
  * Constant flags for byte corruption modes.
  */
-#define CORRUPT_ZERO 1000
-#define CORRUPT_RAND 1001
-#define CORRUPT_ZERO_SEQ 1100
-#define CORRUPT_RAND_SEQ 1101
-#define CORRUPT_DROP 1200       // this will silently drop data from write calls
-
-/**
- * Generate a random double between 0 and 1.0
- */
-#define RAND_FRAC ((double)rand()/(double)RAND_MAX)
+typedef enum {
+    CORRUPT_ZERO = 1000,        // replace bytes at random positions with \0
+    CORRUPT_RAND = 1001,        // replace bytes at random positions with random char value
+    CORRUPT_ZERO_SEQ = 1100,    // replace sequential bytes at the end of the file with \0
+    CORRUPT_RAND_SEQ = 1101,    // replace sequential bytes at the end of the file with random char value
+    CORRUPT_DROP = 1200,        // silently drop bytes at the end of the file
+} corrupt_mode;
```

Comment:
Can you add a comment for why we pick these specific numbers here? Also, is there a reason these codes are not sequential?

Reply:
Sure! I left some gap between the numbers to group sequential corruptions and random corruptions. We may add more corruption modes later for sequential bytes or random bytes.